### PR TITLE
Shows error alert on mobile device 

### LINKED ISSUE #5: Display unsupported message for non-Android mobile users
TODO:

- [x] Try to get the best possible functionality on Android, autoplay will not work, by design.
- [ ] On iOS, and other mobile user agents display a message saying we don't support mobile browsers - since iOS will show videos in full screen, by design. see comments in #38 

Major bugs on Android

screenshot_2015-05-11-07-57-32
screenshot_2015-05-11-07-57-44

### PR DESCRIPTION
**Pull request by FleetAdmiral**
Ignore the previous PR
Rework of #210, closes #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,10 +40,12 @@
         <hgroup>
             <h1>Zen Audio Player</h1>
             <h2 class="color-grey">Listen to YouTube videos, without the distracting visuals.</h2>
+            <h2 class="color-grey" id="mobile-message"></h2>
         </hgroup>
     </header>
     <div id="container">
         <!-- The form -->
+
```
Comment (shakeelmohamed) [inline index.html]: revert this line

File: js/everything.js [hunk 2]
```diff
@@ -431,6 +431,13 @@ function pickDemo() {
 
 $(function() {
     // Keen.io
```
Comment (shakeelmohamed) [inline js/everything.js]: This comment should come before the second `if`

File: js/everything.js [hunk 3]
```diff
@@ -431,6 +431,13 @@ function pickDemo() {
...
+    if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.usersAgent))
+    {
```
Comment (shakeelmohamed) [inline js/everything.js]: this brace should be on the previous line

**Comment by shakeelmohamed** [comment]
Almost there

File: js/everything.js [hunk 4]
```diff
@@ -430,6 +430,12 @@ function pickDemo() {
 }
 
 $(function() {
+    if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.usersAgent)) {
+        $("#container").hide();
+        $("#mobile-message").html("Sorry , we don't support mobile devices.");
```
Comment (shakeelmohamed) [inline js/everything.js]: nit: remove the space before the comma

File: js/everything.js [hunk 5]
```diff
@@ -430,6 +430,12 @@ function pickDemo() {
 }
 
 $(function() {
+    if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.usersAgent)) {
```
Comment (shakeelmohamed) [inline js/everything.js]: `navigator.usersAgent` is undefined, it needs to be `navigator.userAgent`

File: index.html [hunk 6]
```diff
@@ -40,6 +40,7 @@
         <hgroup>
             <h1>Zen Audio Player</h1>
             <h2 class="color-grey">Listen to YouTube videos, without the distracting visuals.</h2>
+            <h2 class="color-grey" id="mobile-message"></h2>
```
Comment (shakeelmohamed) [inline index.html]: also add the `flash` class here

**Comment by shakeelmohamed** [comment]
Just tested with the iOS simulator on OS X, and Chrome's device simulators. Fix those 2 things and I'll merge it

File: index.html (hunk 6 above)
Comment (FleetAdmiral) [inline index.html]: @shakeelmohamed Sorry, i couldn't find the flash class

File: index.html [hunk 7]
```diff
@@ -40,6 +40,7 @@
         <hgroup>
             <h1>Zen Audio Player</h1>
             <h2 class="color-grey">Listen to YouTube videos, without the distracting visuals.</h2>
+            <h2 class="color-grey,hide" id="mobile-message"></h2>
```
Comment (shakeelmohamed) [inline index.html]: classes are space delimited, you also need the `flash` class

**Comment by shakeelmohamed** [comment]
Thanks for sticking with it @FleetAdmiral, welcome to the org!

**Comment by FleetAdmiral** [comment]
@shakeelmohamed Thanks for all the help 👍 . Will stick around for sure!

**Pull request merged**